src/components/routes/chat/ChatPanel.tsx

```diff
@@ -257,9 +260,15 @@ export default function ChatPanel(props) {
 		}
 	</Box>;
 
+	const dateToday = new Date();
+
 	return (
 		<Box className={classes.flexGrow}>
-			<Box className={classes.chatPanel}>
+			<Box className={
+				(dateToday.getMonth() === 9 && dateToday.getDate() >= 30) || (dateToday.getMonth() === 10 && dateToday.getDate() === 1)
```

Comment:
I think there is a small typo here? I think it should be like this: 

(dateToday.getMonth() === 9 && dateToday.getDate() >= 30) && (dateToday.getMonth() === 10 && dateToday.getDate() <= 1)

The current if check will always show the Halloween theme for every day after 30th Oct 

Everything looks good but someone else should check it too, as an enhancement for future (new issue?), it might be better to include the activation days in a themes object, it could get a bit cluttered if all other theme dates are hardcoded into an if statement. For example something like this:

```
themesChatPanel: [
{ startDate: '30-09', endDate: '01-10', isYearly: true, themeName: halloweenChatPanel},
{ startDate: '01-11', endDate: '07-00', isYearly: true, themeName: christmasChatPanel}
]
```

Reply:
The existing check will test if the current date is any date after the 30th **in October** (i.e. October 30th or 31st) or 1st November. The check you've proposed would test if it is October 30th or 31st and November 1st at the same time - which of course wouldn't happen.

If we decide to have more themed events, having some sort of object to hold the dates would definitely be worth it!

Reply:
Oh I see, I misread it, my bad